packages/clay-date-picker/src/index.tsx

```diff
@@ -554,32 +557,34 @@ const DatePicker = React.forwardRef<HTMLInputElement, IProps>(
 		 * is no date selected.
 		 */
 		const handleDotClicked = () => {
+			const currentTime = defaultMonth || new Date();
```

Comment:
defaultMonth favored over _now_

Reply:
Can't we just use `initialMonthInternal` here? If not because it's not always getting now correctly, why not use a function instead of `initialMonthInternal`.
```
const getDefaultMonth = useCallback(() => defaultMonth ?? initialMonth ?? new Date(),[defaultMonth, initialMonth]);
```
This would keep us from duplicating logic and still handle the fallback to the deprecated prop as well.